src/components/BasicAccordion/BasicAccordion.module.scss

```diff
@@ -0,0 +1,8 @@
+.accordion {
+  border-top: 1.5px solid #e0e0e0;
```

Comment:
tailwind css is not used here, we are only supposed to use .scss file in case of dynamic styling.